memgpt/cli/cli_config.py

```diff
@@ -116,24 +133,51 @@ def configure_model(config: MemGPTConfig, model_endpoint_type: str):
             ).ask()
             model = None if len(model) == 0 else model
 
+        default_model = config.model if config.model and config.model_endpoint_type == "vllm" else ""
+
         # vllm needs huggingface model tag
         if model_endpoint_type == "vllm":
-            default_model = config.model if config.model and config.model_endpoint_type == "vllm" else ""
-            model = questionary.text(
-                "Enter HuggingFace model tag (e.g. ehartford/dolphin-2.2.1-mistral-7b):",
-                default=default_model,
-            ).ask()
-            model = None if len(model) == 0 else model
-            model_wrapper = None  # no model wrapper for vLLM
```

Comment:
FYI removed this, not sure why we weren't asking for a wrapper w/ vLLM

Basically as long as the backend is not fully-OpenAI compatible (including function call compatible) then we should specify a wrapper (because in that case, we'll be doing our own prompt formatting then routing the request to a `/completions` API instead of a `/chat/completions` API.